packages/i18n/src/index.js

```diff
@@ -1,172 +1,7 @@
 /**
- * External dependencies
+ * Internal dependencies
```

Comment:
For a reexport-only module, this docblock doesn't seem to make much sense. If I remove it, the linter doesn't complain.